client/entrypoints/payment-request-settings/__tests__/payment-request-button-preview.test.js

```diff
@@ -4,8 +4,12 @@ import { useStripe } from '@stripe/react-stripe-js';
 import PaymentRequestsButtonPreview from '../payment-request-button-preview';
 import { shouldUseGooglePayBrand } from '../utils/utils';
 
-jest.mock( '@wordpress/a11y', () => ( {
-	...jest.requireActual( '@wordpress/a11y' ),
+// We need to mock the actual module being used by `<Notice />` in the `@wordpress/components` module
+const realPathToA11yModule =
+	'../../../../node_modules/@wordpress/components/node_modules/@wordpress/a11y';
```

Comment:
Nit: `@wordpress/components/node_modules/@wordpress/a11y` should work too. It's a bit shorter.

Reply:
done https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2280/commits/14bc2a4ae07d76a246507f9c03d568e7ca81eab4